pkg/kv/dynamodb/store.go

```diff
@@ -351,6 +382,9 @@ func (s *Store) DropTable() error {
 	_, err := s.svc.DeleteTable(&dynamodb.DeleteTableInput{
 		TableName: &s.params.TableName,
 	})
+	if err != nil {
+		dynamoFailures.WithLabelValues("DeleteTable").Inc()
```

Comment:
This is an internal function for testing. Not sure we need to collect metrics on it.

Reply:
true, will remove it